genie-web/src/integTest/java/com/netflix/genie/web/data/services/impl/jpa/JpaPersistenceServiceImplJobsIntegrationTest.java

```diff
@@ -1224,6 +1226,25 @@ void canGetJobHostInformation() {
         Assertions.assertThat(randomInfo.getTotalMemoryUsed()).isEqualTo(0L);
     }
 
+    @Test
+    @DatabaseSetup("persistence/jobs/init.xml")
```

Comment:
you may want to break this `setup` away from this global init as it make it fragile if someone else accidently changes that. I've been trying to scope them to specific tests -> specific init files that way one change to support test A doesn't break tests B, C, D

Reply:
This `init.xml` is used and shared by numerous other tests in the same file. For example get/set status.
They're all very similar and I think it's appropriate they share the same test data.
If i was to clone it, it would be an identical copy of the existing one, and next time something changes one has to go and update multiple identical files.